openedx/features/course_experience/url_helpers.py

```diff
@@ -175,6 +217,28 @@ def get_learning_mfe_home_url(
     return mfe_link
 
 
+def get_catalog_mfe_course_about_url(
```

Comment:
Why do we need this new function? Why can't we use the `get_link_for_about_page` function that already exists and that you have already updated to work with the catalog MFE in https://github.com/openedx/edx-platform/pull/37342